Dockerfile

```diff
@@ -108,6 +110,15 @@ RUN pip install -r requirements.txt -r requirements-dev.txt
 # Jenkins runs the containers with the `-u 1000:1000` option, not as root.
 RUN chown -R +1000:+1000 /venv
 
+# spead2_net_raw is needed so that qualification tests can use ibverbs
+RUN SPEAD2_VERSION=$(grep ^spead2== requirements.txt | cut -d= -f3) && \
+    cd /tmp && \
+    wget https://github.com/ska-sa/spead2/releases/download/v$SPEAD2_VERSION/spead2-$SPEAD2_VERSION.tar.gz && \
+    tar -zxf "spead2-$SPEAD2_VERSION.tar.gz" && \
+    cd "spead2-$SPEAD2_VERSION/src" %&& \
+    gcc -Wall -O2 -o /usr/local/bin/spead2_net_raw spead2_net_raw.cpp -lcap && \
+    setcap cap_net_raw+p /usr/local/bin/spead2_net_raw
+
```

Comment:
For tidyness, I'd also suggest removing the source code at the end of this step, so that it doesn't leave a random file lying around in the container.

Reply:
I'd kind of hoped that by virtue of being in `/tmp`, that would be taken care of.